src/pages/workspace/members/WorkspaceInviteMessageComponent.tsx

```diff
@@ -146,6 +162,14 @@ function WorkspaceInviteMessageComponent({
             return;
         }
 
+        // If backTo is provided and it's not the members page, navigate back to it
+        if (backTo && !(backTo as string).endsWith('members')) {
```

Comment:
I think we should check `'expenses-from'` ROUTE only and goBack to avoid hidden bugs